docs/spec.json

```diff
@@ -1617,6 +1634,16 @@
       },
       "x-go-package": "github.com/gotify/server/model"
     },
+    "RawMessage": {
+      "description": "It implements Marshaler and Unmarshaler and can\nbe used to delay JSON decoding or precompute a JSON encoding.",
+      "type": "array",
```

Comment:
What information will be lost?

Reply:
I got that there was once a case I need to dealt a json where I would need the sequence of the keys occured in an object. Like for `{"a": 1, "b": 2}` I need to get `[]string{"a", "b"}`. This is possible in JS and is also possible in golang with a custom unmarshaler, but unmarshaling it to an `interface{}` would cause this piece of information to be lost.

This is not very important though, I think it can be ignored.

Reply:
Yeah, lets ignore it.